gunrock/app/sm/sm_app.cuh

```diff
@@ -108,6 +109,9 @@ cudaError_t RunTests(
             h_subgraphs, ref_subgraphs, false);
     }
 
+    UseParameters_test(parameters);
+    parameters.Set("num_subgraphs", num_subgraphs);
```

Comment:
You need to add this in UseParameters first before setting it: https://github.com/gunrock/gunrock/blob/9fe1ef2418ee3a215b2ac439a36c4f32f766c55b/gunrock/app/color/color_app.cu#L42-L45

Also, can you please use `num-subgraphs` and not underscore? All json variables use minus/dash instead

Reply:
Also, can you please test it with --json to see if the num-subgraphs actually show up?